### ISSUE TITLE
[Non-critical] Automatic mode and container name prefix changed behaviour.

### ISSUE THREAD

**Issue by crssi** [stoically/temporary-containers#235]
- Temporary Containers Version: 0.93
- Firefox Version: 66.0.2
- Operating System + Version: Windows 10
- Non-default Options/Preferences in Temporary Containers:
Automatic Mode: Enabled
Container Name Prefix: %domain% #
- Other installed Add-ons + Version: None
Go to FF options -> Home and set "New Windows and Tabs" to "Blank Page"

### Actual behavior
Open new tab. The container name gets the name as a number.
Navigate to some domain.
Container name is still a number.

### Expected behavior
Open new tab. It should not open in a container.
Navigate to some domain, which will force tab to open in a container.
Container name is set to the initial domain as a prefix with the number as suffix.

### Notes
This is a new behaviour in the latest TC. Few days ago this worked fine.
The right behaviour workaround is to set `Automatic Mode: Don't reopen New Tabs in Temporary Containers but instead on navigation.` in advanced settings.

I have duckduckgo as default search engine and TC settings as follows (addition to default):
```
General
  Automatic Mode: Check
  Container Name Prefix: %domain% #
  Random Container Color: Check
  Container Number: Reuse available numbers
  Delete no longer needed Temporary Containers: 2 minutes after the last tab in it closes
  Click [Save]
Isolation
  Per Domain
	Domain pattern: *
	  Always open in new Temporary Containers: Enabled
	  Click [Add or Edit]
	Domain pattern: duckduckgo.com
	  Always open in new Temporary Containers: Enabled
	  Mouse Clicks on Links should open new Temporary Containers [Left Mouse]: If the clicked Link Domain does not match the active Tabs Domain (Subdomains won't get isolated)
	  Click [Add or Edit]
  Global
	Mouse Clicks on Links should open new Temporary Containers
	  Middle Mouse: If the clicked Link Domain does not match the active Tabs Domain (Subdomains won't get isolated)
	Click [Save]
Advanced
	General
	  Automatic Mode: Don't reopen New Tabs in Temporary Containers but instead on navigation.
  Click [Save]
```
That said, I would say this issue is a **non-critical** and if it does not bother you, then leave it and close this issue. :wink: 

Cheers

**Comment by stoically** [comment]
Thanks for the report. With the fix introduced in #146 TC now also reopens new tabs if set to "Blank Page" with the default "Reopen new tabs" automatic mode preference - before that fix it was essentially the same behavior as with the automatic mode preference "Don't reopen new tabs", because new about:blank tabs weren't reopened in TCs. If the new tab is reopened instantly into a TC, there's no domain it could use for the %domain% prefix, because there was no navigation yet. So, if you want your expected behavior, you'd have to configure "Don't reopen". Hope that helps / explains it.

**Comment by crssi** [comment]
I suspected accurate and the "workaround" is just fine, since now it works actually more accurate... I guess.

Thank you and cheers
